src/index.ts

```diff
@@ -1,28 +1,24 @@
 import { getInput, setFailed, warning } from '@actions/core';
 import { context, GitHub } from '@actions/github';
 
-import * as checkSuite from './eventHandlers/checkSuite';
-import * as pullRequest from './eventHandlers/pullRequest';
-import * as push from './eventHandlers/push';
+import {
+  checkSuiteHandle,
+  pullRequestHandle,
+  pushHandle,
+} from './eventHandlers';
 
 const GITHUB_TOKEN = getInput('GITHUB_TOKEN');
 
 const octokit = new GitHub(GITHUB_TOKEN);
 
 const main = async (): Promise<void> => {
-  console.log(
-    context.eventName,
-    context.repo,
-    JSON.stringify(context.payload, undefined, 2),
-  );
-
   switch (context.eventName) {
     case 'check_suite':
-      return checkSuite.handle(octokit);
+      return checkSuiteHandle(octokit);
     case 'pull_request':
-      return pullRequest.handle(octokit);
+      return pullRequestHandle(octokit);
     case 'push':
-      return push.handle(octokit);
+      return pushHandle(octokit);
     default:
       warning(`Unknown event ${context.eventName}, skipping.`);
   }
```

Comment:
The functions repository relies on status as well as on check_suite events. Statuses are being generated by Codecov and are usually the last events that fire in the chain. Shouldn't we have a handler for status changes as well?

Reply:
we definitely should, this is good insight. However I consider it out of the scope of this particular PR. I have created an issue for this here: https://github.com/ridedott/auto-merge-action/issues/16